communication/makefile

```diff
@@ -7,6 +7,6 @@ TARGET_TYPE = a
 
 BUILD_PATH_EXT=$(COMMUNICATION_BUILD_PATH_EXT)
 
-DEPENDENCIES = hal dynalib services wiring
+DEPENDENCIES = hal dynalib services wiring system
```

Comment:
We've been very conscious to avoid coupling `communication` with `system`. For any external notifications/behaviour, please add a callback to avoid a direct coupling.